tests/test_analysis_tools.py

```diff
@@ -10,14 +10,14 @@
 fname = "tests/samples/nano_dy.root"
 eagerevents = NanoEventsFactory.from_root(
     {os.path.abspath(fname): "Events"},
-    schemaclass=NanoAODSchema.v6,
+    schemaclass=NanoAODSchema,
```

Comment:
Does the `.v6` specialization still work but is just not needed?

Reply:
Right now the `.vX` specializations are a little broken in their original implementation. The functional sub-classing thing you were doing doesn't quite fit into the daskiness.